content/docs/enterprise/external-data/geoip.mdx

```diff
@@ -61,14 +114,20 @@ allow:
     - record:
         field: country
         is: US
-        type: pomerium.io/IP2Location
+        type: pomerium.io/GeoIP
 ```
 
 </TabItem>
 </Tabs>
 
-This policy looks at the `country`
+This policy looks at the `country`, and allows access only from IP addresses purportedly in the US.
 
 ## Reference
 
-[Docker Compose]: https://docs.docker.com/compose/
\ No newline at end of file
+- The ip2location datasource expects one argument when running the binary, the path to the GeoIP database.
+- It has an optional flag `--address`, to change the listening port from the default `:8080`.
```

Comment:
@calebdoxsey This was my attempt to make a reference section based on the output of the help flag. Please add any additional flags or arguments.

Reply:
ok. I don't know of any additional flags or arguments.